src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -35,55 +35,81 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     ];
 
     if (!ngEntryPoint.isSecondaryEntryPoint) {
-      const assetFiles: string[] = [];
-
       // COPY ASSET FILES TO DESTINATION
       spinner.start('Copying assets');
 
       try {
-        for (const asset of [...ngPackage.assets, 'LICENSE', '**/README.md']) {
-          let assetFullPath = path.join(ngPackage.src, asset);
-
+        type AssetCopyTask = [srcPath: string, destPath: string];
```

Comment:
Can you move this outside of the method? Maybe below the imports?

Reply:
This type has been removed in the new implementation